target/arc/translate.c

```diff
@@ -106,6 +106,36 @@ static void gen_gotoi_tb(DisasContext *ctx, int n, target_ulong dest)
     }
 }
 
+static void
+arc_cpu_release_llockscond_locks(ARCCPU *cpu) {
```

Comment:
Missing comment on what the function should do.

Reply:
Missing comment added